### PR TITLE
Add Icons to Menu Tab (#15)

### PR DESCRIPTION
**Pull request by goccert25**
Adds an additional library to manage icons. More found here: https://github.com/oblador/react-native-vector-icons. 
Icon will changes based on platform. Uses native iOS Icons for iOS and material design Icons for android. All code for menu tab overlay can be found in CustomTabBarOverlay.js.
Styling still needs to be done to match bitcamp design specs.

### CODE REVIEW

**Comment by jsjqian** [comment]
LGTM. I don't see any major issues and it works on Android.

File: CustomTabBarOverlay.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  Text,
+  View,
+  Animated,
+  TouchableOpacity,
+} from 'react-native';
+
+import { colors } from './shared/styles';
+import Icon from 'react-native-vector-icons/Ionicons';
+
+const CustomTabBarOverlay = React.createClass({
```
Comment (jdai8) [inline CustomTabBarOverlay.js]: We should probably be consistent and use `class` instead of `React.createClass`

File: CustomTabBarOverlay.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    });
+
+    return <View style={[styles.tabs, this.props.style, ]}>
+      {this.props.tabs.map((tab, i) => {
+        return <TouchableOpacity key={tab} onPress={() => this.props.goToPage(i)} style={styles.tab}>
+          <Icon
+            name={tab}
+            size={30}
+            color={this.props.activeTab === i ? colors.bitcampOrange : 'rgb(204,204,204)'}
+            ref={(icon) => { this.tabIcons[i] = icon; }}
```
Comment (jdai8) [inline CustomTabBarOverlay.js]: Do we ever use this ref?

File: CustomTabBarOverlay.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    const eachTabWidth = {
+      width: containerWidth/numberOfTabs
+    };
+    const left = this.props.scrollValue.interpolate({
+      inputRange: [0, 1, ], outputRange: [0,  containerWidth / numberOfTabs, ],
+    });
+
+    return <View style={[styles.tabs, this.props.style, ]}>
+      {this.props.tabs.map((tab, i) => {
+        return <TouchableOpacity key={tab} onPress={() => this.props.goToPage(i)} style={styles.tab}>
```
Comment (jdai8) [inline CustomTabBarOverlay.js]: 80 character lines

File: MenuTab.js [hunk 4]
```diff
@@ -10,26 +10,48 @@ import IntroScene from './scenes/IntroScene';
...
 
     return (
-      <ScrollableTabView tabBarPosition={menuTabPosition}>
-        <IntroScene tabLabel="Intro"/>
-        <ScheduleScene tabLabel="Schedule" />
-        <MapScene tabLabel="Map" />
-        <CompanyInfoScene tabLabel="Company Info" />
-        <MentorsScene tabLabel="Mentors" />
+      <ScrollableTabView
+      tabBarPosition={menuTabPosition}
```
Comment (jdai8) [inline MenuTab.js]: Can we indent these please? 😸😺

**Review by jdai8 — CHANGES_REQUESTED** [review]
LGTM, made some small code comments.

**Comment by jdai8** [comment]
Some things we might want to think about for later:
- Finalize the styling of the tab bar (color, spacing, night mode, etc). I kinda think we should use [this](http://facebook.github.io/react-native/releases/0.34/docs/tabbarios.html) for iOS, though that's mostly a styling thing we can decide later.
- This PR also includes a placeholder schedule, but Andy's working on a "better" one.

**Pull request merged**